[TC-6941] Basic delivery line logistics (#26)
tradecloud/tradecloud-docs-api-v2: buyer/receive/README.md

The`deliveryOverdue`feature is planned and API and documentation may change.
{% endhint %}

> * `status.processStatus`: is the aggregate of all lines [process status](#process-status).

Reviewer: ```suggestion
* `status.processStatus`: is the aggregate of all lines' [process statuses](#process-status).
```

> * `status.logisticsStatus`: is the aggregate of all lines [logistics status](#logistics-status).

Reviewer: ```suggestion
* `status.logisticsStatus`: is the aggregate of all lines' [logistics statuses](#logistics-status).
```

* `buyerLine`: the buyer part of the order line, see [Buyer line part](./#buyer-line-part).
* `supplierLine`: the supplier part of the order line, see [Supplier line part](./#supplier-line-part).
* `confirmedLine`: the order line as agreed between buyer and supplier, see [Confirmed line](./#confirmed-line).

> * `deliverySchedule`: the aggregated delivery schedule line with logistics info, see [Delivery Schedule](./#delivery-schedule) below.

Reviewer: ```suggestion
* `deliverySchedule`: the aggregated delivery schedule lines with logistics info, see [Delivery Schedule](./#delivery-schedule) below.
```

* `indicators.deliveryOverdue` is true when the order line is overdue.

{% hint style="warning" %}
The`deliveryOverdue`feature is planned and API and documentation may change.
{% endhint %}

> * `status.processStatus`: the order line [process status](#process-status).

Reviewer: ```suggestion
* `status.processStatus`: the order line's [process status](#process-status).
```

> * `status.logisticsStatus`: the order line [logistics status](#logistics-status).

Reviewer: ```suggestion
* `status.logisticsStatus`: the order line's [logistics status](#logistics-status).
```

* `deliverySchedule`: agreed delivery schedule, see below
* `prices`: agreed prices, see below

## Delivery schedule

> `deliverySchedule`: the proposed or confirmed planned delivery schedule.

Reviewer: ```suggestion
`deliverySchedule`: the proposed or confirmed delivery schedule.
```

* `deliverySchedule.position`: the optional position in the delivery schedule. Not to be confused with the `line.position`
* `deliverySchedule.date`: the delivery date of this delivery schedule position. Date has ISO 8601 date `yyyy-MM-dd` format. See also [Standards](../../api/standards.md).
* `deliverySchedule.quantity`: the quantity of this delivery schedule position. Quantity has a decimal `1234.56` format with any number of digits.

### Logistics fields

These additional logistics fields are only available in the order line level delivery schedule:

> * `deliverySchedule.status`: the optional delivery line [logistics status](#logistics-status).

Reviewer: ```suggestion
* `deliverySchedule.status`: the optional delivery line's [logistics status](#logistics-status).
```

> * `deliverySchedule.eta`: The optional logistics estimated time of arrival local date (without time zone). Date has ISO 8601 date `yyyy-MM-dd` format.

Reviewer: ```suggestion
* `deliverySchedule.eta`: The optional logistics estimated time of arrival (local date without time zone). Date has ISO 8601 date `yyyy-MM-dd` format.
```
